mflowgen/Tile_MemCore/construct.py

```diff
@@ -66,9 +66,10 @@ def construct():
   constraints          = Step( this_dir + '/constraints'                           )
   gen_sram             = Step( this_dir + '/../common/gen_sram_macro'              )
   custom_init          = Step( this_dir + '/custom-init'                           )
+  custom_genus_scripts = Step( this_dir + '/custom-genus-scripts'                  )
+  custom_flowgen_setup = Step( this_dir + '/custom-flowgen-setup'                  )
   custom_lvs           = Step( this_dir + '/custom-lvs-rules'                      )
   custom_power         = Step( this_dir + '/../common/custom-power-leaf'           )
-  custom_timing_assert = Step( this_dir + '/../common/custom-timing-assert'        )
```

Comment:
Why are we getting rid of this?

Reply:
This assertion doesn't make sense anymore with the new mem tile ports. It expects there to be a `clk_pass_through_out` port.

Reply:
Ah, that's right, since this is for the mem tile only.